c10/util/Synchronized.h

```diff
@@ -28,7 +26,7 @@ class Synchronized final {
  public:
   Synchronized() = default;
   Synchronized(T const& data) : data_(data) {}
-  Synchronized(T&& data) : data_(data) {}
+  Synchronized(T&& data) : data_(std::move(data)) {}
```

Comment:
Shouldn't this be a perfect forward here?

Reply:
We can use perfect forward here and remove the above constructor.

Reply:
@Skylion007  No, this is a move constructor, not universal reference. The fix is correct.